src/server.rs

```diff
@@ -73,7 +75,18 @@ impl Server {
     /// migrations required, create various guarded pool objects, and start the SQS handler
     /// (unless we're testing)
     pub fn start(rocket: rocket::Rocket) -> Result<rocket::Rocket, HandlerError> {
-        db::run_embedded_migrations(rocket.config())?;
+        // tests on circle can re-run the embedded migrations. this can fail because of
```

Comment:
Sounds like this may not be necessary with the additions to the migration sql?

Reply:
Honestly, it's odd. I updated the SQL to be smarter, but would sometimes hit an error anyway, usually in testing. I think it might be from the multi-thread startup adding a bit of complexity. I'd like to leave this here just in case we do hit any problems.

Reply:
You probably do need to force `RUST_TEST_THREADS=1`, it is missing here from the original travis config. diesel definitely seems to break without it (I can't recall if it breaks during the migration runner or elsewhere).

Reply:
crap, yeah, you're right. I'll file a follow up issue.